apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTABanner>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Aligned center
+
+The content alignment can be changed using the `align` prop on the root `<ComparisonTable>`. This can be configured to either `start` or `center`.
+
+```jsx live
+<CTABanner align={'center'}>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Optional border
+
+A border can be provided using the `hasBorder` prop on the root `<ComparisonTable>`. This will render a border around the CTABanner component giving separatoin between the foreground and background. Especially when there is no shadow present on the background.
+
+```jsx live
+<CTABanner hasBorder>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Optional background
+
+A background can be removed using the `hasBackground` prop on the root `<ComparisonTable>`. This will render the CTABanner component without a background allowing it to blend in with its parent.
+
+```jsx live
+<CTABanner hasBackground={false}>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Optional shadow
+
+The shadow can be removed using the `hasShadow` prop on the root `<ComparisonTable>`. This will render the CTABanner component without the background shaow.
```

Comment:
typo's and another reference to `ComparisonTable`

```suggestion
The shadow can be removed using the `hasShadow` prop on the root `CTABanner`. This will render the component without the background shadow.
```

```